src/app/container/launch/tool-launch.service.spec.ts

```diff
@@ -41,14 +41,18 @@ describe('LaunchService', () => {
 
     it('should getCWLString', inject([ToolLaunchService], (service: ToolLaunchService) => {
         expect(service.getCwlString('quay.io/a/b', 'c'))
-            .toContain('cwltool --non-strict');
+            .toContain('cwl-runner --non-strict');
         expect(service.getCwlString('quay.io/a/b', 'c'))
             .toContain('api/ga4gh/v1/tools/quay.io%2Fa%2Fb/versions/c/plain-CWL/descriptor Dockstore.json');
         expect(service.getCwlString('quay.io/a/b/d', 'c'))
             .toContain('api/ga4gh/v1/tools/quay.io%2Fa%2Fb%2Fd/versions/c/plain-CWL/descriptor Dockstore.json');
     }));
     it('should getConsonanceString', inject([ToolLaunchService], (service: ToolLaunchService) => {
-        expect(service.getConsonanceString('a/b', 'latest')).toContain(
-            'consonance run --tool-dockstore-id a/b:latest --run-descriptor Dockstore.json --flavour <AWS instance-type>');
+        expect(service.getAltCwlString1('quay.io/briandoconnor/dockstore-tool-bamstats', '1.25-11')).toContain(
+            'cwltool --non-strict quay.io/briandoconnor/dockstore-tool-bamstats:1.25-11 Dockstore.json');
+    }));
+    it('should getAlternateStrings', inject([ToolLaunchService], (service: ToolLaunchService) => {
+        expect(service.getAltCwlString2('quay.io/briandoconnor/dockstore-tool-bamstats', '1.25-11')).toContain(
+            'cwltool --make-template quay.io/briandoconnor/dockstore-tool-bamstats:1.25-11 > input.yaml');
```

Comment:
Is cwltool correct here?

Reply:
The issue mentioning the new section shows cwltool as the example